microsoft/lisa
# DPDK: Extend netvsc pmd support

New 6 testcases for Netvsc pmd:

1. VERIFY-DPDK-BUILD-AND-NETVSCPMD-TEST
2. VERIFY-SRIOV-FAILSAFE-FOR-DPDK-NETVSCPMD
3. VERIFY-DPDK-FAILSAFE-DURING-TRAFFIC-NETVSCPMD
4. PERF-DPDK-FWD-PPS-NETVSCPMD
5. PERF-DPDK-SINGLE-CORE-PPS-NETVSCPMD
6. PERF-DPDK-MULTICORE-PPS-NETVSCPMD
